modules/manage/pages/cluster-maintenance/partition-recovery.adoc

```diff
@@ -0,0 +1,31 @@
+= Forced partition recovery
+:description: Recover a single partition using the Admin API.
+
+You can use the Redpanda Admin API to recover a partition that appears to be stuck (for example, does not accept writes). This can occur when the partition replicas have lost glossterm:Raft[] consensus, for instance if brokers in a Raft group fail, preventing the group from reaching a majority and electing a new leader. 
+
+Forced partition recovery is carried out by promoting the best available replica to leader, allowing the partition to become available for produce and consume. Typically, Redpanda chooses the replica with the highest offset. However, this allows some potential data loss on the partition if the best available replica is out of sync, resulting in an unclean leader election. Forced partition recovery should only be used when brokers have failed beyond recovery to the point that the remaining replicas cannot form a majority. 
+
+If you want to instead force recover partitions in bulk on a failed broker, use nodewise recovery.  
```

Comment:
I'd rephrase this -- it's not so much when you want to force recover partitions as needing to do a recovery with whole dead nodes. Nodewise partition recovery force-recovers a node, letting partitions which have a majority be rebuilt while starting fresh partitions where there is no majority.